components/permit-holders/DoctorInformationCard.tsx

```diff
@@ -3,10 +3,12 @@ import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; //
 import { Physician } from '@lib/graphql/types'; // Physician type
 import EditDoctorInformationModal from '@components/requests/modals/EditDoctorInformationModal'; // Edit doctor information modal component
 import PreviousDoctorsInformationModal from '@components/permit-holders/modals/PreviousDoctorsInformationModal'; // Previous Doctors' Information Modal
+import { PreviousPhysicianData } from '@pages/admin/permit-holder/[permitHolderId]';
 
 type DoctorInformationProps = {
-  physician: Physician;
+  physician?: Physician;
   readonly isUpdated?: boolean;
+  readonly previousPhysicianData?: PreviousPhysicianData[];
```

Comment:
```suggestion
  readonly previousPhysicianData: PreviousPhysicianData[];
```
We should just make this an array. If there are no previous physicians, our API should return an empty array rather than `undefined`